messaging/src/main/java/ids/messaging/endpoint/MessageController.java

```diff
@@ -241,6 +267,13 @@ public ResponseEntity<MultiValueMap<String, Object>> handleIDSMessage(
         }
     }
 
+    private void logSendResponse(final MultiValueMap<String, Object> responseAsMap) {
+        if (Boolean.TRUE.equals(logResponse)) {
+            final var header = responseAsMap.get(MultipartDatapart.HEADER.toString()).toString();
+            log.info("Send response header: {}", header);
```

Comment:
Technically this is because when sending a message the header itself was explicitly selected for logging, whereas with incoming messages everything that arrives is simply logged. It is difficult to change that, because the logging of incoming messages happens directly after receipt and the incoming message should not be passed through e.g. the serializer first to have a nicer formatting, because the seralizer can already throw exceptions. Will see if something can be changed.